src/notes/patchV3.6.7.md

```diff
@@ -3,75 +3,75 @@
   - Beyond the level transitions between the Wan Chai Market and the Lucky Money
   - Beyond the door to the Vandenberg Computer area from Cmd
-- Removed collision from teleporters so they don't block pawns or items
-- The Bingo Viewer now has an option to always keep the window on top (Under the Display menu)
-- Looking at Weeping Anna will no longer interrupt her death animation
-- For GMDX we now set Hardcore mode when playing on combatDifficulty 4 or higher
-- Fixed randomizing some of GMDX's invisible objects
-- Fixed bug with conversations referencing player's kill counts in M01 and M02
-- Added a timed race to Vandenberg Tunnels (in both the forward and backward directions):  See how fast you can go!
-- UNATCO Troopers that appear in Mission 3, after meeting Lebedev, will no longer hate seeing carcasses (which typically makes them hate the player)
-- The bingo goal for returning to UNATCO HQ in the boat after Mission 2 no longer appears unless the bingo board spans more than 1 mission.
-- Possible fix for some timed races (eg. Catacombs or Reverse Catacombs) where the Mastodon messages were reporting the player losing over 9000% health.
+- Removed collision from teleporters so they don't block pawns or items.
+- The Bingo Viewer now has an option to always keep the window on top (under the Display menu).
+- Looking at Weeping Anna will no longer interrupt her death animation.
+- For GMDX we now set Hardcore mode when starting a game with combat difficulty 4 or higher (Realistic and Impossible).
+- Fixed randomizing some of GMDX's invisible objects.
+- Fixed a bug with conversations referencing the player's kill counts in Mission 1 and Mission 2.
+- Added a timed race to Vandenberg Tunnels (in both the forward and backward directions): See how fast you can go!
+- UNATCO Troopers that appear in Mission 3, after meeting Lebedev, will no longer hate seeing carcasses (which typically makes them hate the player).
+- The bingo goal for returning to UNATCO HQ in the boat after Mission 2 no longer appears unless the bingo board spans more than 1 Mission.
+- Possible fix for some timed races (e.g. Catacombs or Reverse Catacombs) where the Mastodon messages were reporting the player losing over 9000% health.
 - Fixed installer failing with Documents path issues
-- The key for the cabinet in the Mission 2 Free Clinic is now adjusted more reliably and should now work properly in Vanilla? Madder!
-- Add two new bingo goals for interacting with Joe Greene in Mission 2 and 4.
-- The description for the Energy Shield augmentation now mentions that it works against explosive damage as well.  This is vanilla behaviour, so just makes the description more accurate to its functionality.
+- The key for the cabinet in the Mission 2 Free Clinic is now adjusted more reliably and should now work properly in Vanilla? Madder!.
+- Two new bingo goals for interacting with Joe Greene in Mission 2 and Mission 4.
```

Comment:
Being new goals, these should actually probably be listed under the bullet points of new goals further down